frontend/src/metabase/components/form/FormWidget.jsx

```diff
@@ -3,6 +3,7 @@ import React, { forwardRef } from "react";
 
 import { PLUGIN_FORM_WIDGETS } from "metabase/plugins";
 
+import CategoryFieldPicker from "metabase/core/components/FormCategoryInput";
```

Comment:
Should we import it here as `FormCategoryInput` and rename the component in the file?

Reply:
It doesn't matter at all because a few commits down the road this line gets deleted 😁